scripts/badger/process_bribes_graviaura.py

```diff
@@ -0,0 +1,88 @@
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# only set to true when actually ready to post and exec on mainnet
+COW_PROD = False
+
+# artificially create slippage on the quoted price from cowswap
+COEF = 0.9825
+
+# time after which cowswap order expires
+DEADLINE = 60 * 60 * 3
+
+# percentage of the bribes that is used to buyback $badger
+# ref: https://forum.badger.finance/t/bip-95-graviaura-regulations/5716
+BADGER_SHARE = 0.25
+
+SAFE = GreatApeSafe(r.badger_wallets.techops_multisig)
+SAFE.init_badger()
+SAFE.init_cow(prod=COW_PROD)
+PROCESSOR = SAFE.contract(r.aura_bribes_processor)
+
+# tokens involved during processing
+WETH = interface.IWETH9(r.treasury_tokens.WETH, owner=SAFE.account)
+BADGER = interface.ERC20(r.treasury_tokens.BADGER, owner=SAFE.account)
+AURA = interface.ERC20(r.treasury_tokens.AURA, owner=SAFE.account)
+
+
+def claim_and_sell_for_weth():
+    bribes_dest = GreatApeSafe(PROCESSOR.address)
+    bribes_dest.take_snapshot(r.bribe_tokens_claimable.values())
+
+    claimed = SAFE.badger.claim_bribes_hidden_hands()
+
+    for addr, mantissa in claimed.items():
+        order_payload, order_uid = SAFE.badger.get_order_for_processor(
+            sell_token=SAFE.contract(addr),
+            mantissa_sell=mantissa,
```

Comment:
```suggestion
            mantissa_sell=int(mantissa),
```

Reply:
`mantissa` is `str` here

Reply:
good catch!

Reply:
even tho, i think will be resolved [here](https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/ape_api/cow.py#L53) also along the execution of the code